README.id-ID.md

```diff
@@ -67,22 +67,22 @@ Ada dua buah perbedaan antara Nano ID dan UUID v4:
 
 ```rust
 $ node ./test/benchmark.js
-crypto.randomUUID         28,398,328 ops/sec
```

Comment:
Something wrong happened with rebasing. Seems like it is easier to create another PR.

Or give me your email on GitHub and I will copy your changges manually.